shared/src/main/kotlin/com/egm/stellio/shared/util/QueryUtils.kt

```diff
@@ -176,18 +179,27 @@ private fun transformQQueryToSqlJsonPath(
         jsonb_path_exists(#{TARGET}#,
             '$."${mainAttributePath[0]}"."$NGSILD_PROPERTY_VALUE".$jsonTrailingPath.**{0 to 1}."$JSONLD_VALUE" ? 
                 (@ $operator ${'$'}value)',
-            '{ "value": $value }')
+            '{ "value": ${value.replace(
```

Comment:
You use this in multiple places you may want to create an escapeQuote() function in shared/../QueryUtils or search-service/../DBQueryUtils.